src/m365/spo/commands/web/web-roleassignment-remove.spec.ts

```diff
@@ -280,4 +352,48 @@ describe(commands.WEB_ROLEASSIGNMENT_REMOVE, () => {
       }
     });
   });
+
+  it('removes role assignment from web using entraGroupId', async () => {
+    sinon.stub(entraGroup, 'getGroupById').withArgs(graphGroup.id).resolves(graphGroup);
+    sinon.stub(spo, 'ensureEntraGroup').withArgs('https://contoso.sharepoint.com', graphGroup).resolves(entraGroupResponse);
+
+    sinon.stub(request, 'post').callsFake(async (opts) => {
+      if ((opts.url as string).indexOf('_api/web/roleassignments/removeroleassignment(principalid=\'11\')') > -1) {
```

Comment:
in general we should avoid using `indexOf` and instead use exact equal to url using `===`